### PR TITLE
Add API call with authorization

### PR DESCRIPTION
**Pull request by KovalenkoAndrii**
@TanyaKalistruk @AndriiUngurian

### CODE REVIEW

File: MyBooking/Controllers/AuthorizationController.cs [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        /// POST Api/Auth
+        [HttpPost]
+        public ActionResult<UserModel> SignIn([FromBody]CredModel credModel)
+        {
+            User user = credRepository.GetByEmailAndPassword(credModel.UserEmail, credModel.UserPassword);
+
+            if (user == null)
+                return BadRequest();
+
+            return new UserModel(user);
```
Comment (TanyaKalistruk) [inline MyBooking/Controllers/AuthorizationController.cs:31]: So are you expect that every API call should send you creds?

File: MyBooking/Controllers/AuthorizationController.cs [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        }
+
+        /// POST Api/Auth
+        [HttpPost]
+        public ActionResult<UserModel> SignIn([FromBody]CredModel credModel)
+        {
+            User user = credRepository.GetByEmailAndPassword(credModel.UserEmail, credModel.UserPassword);
+
+            if (user == null)
+                return BadRequest();
```
Comment (TanyaKalistruk) [inline MyBooking/Controllers/AuthorizationController.cs]: Maybe Unauthorized?

File: MyBooking/Controllers/AuthorizationController.cs (hunk 1 above)
Comment (TanyaKalistruk) [reply MyBooking/Controllers/AuthorizationController.cs:31]: Discussed

File: MyBooking/Controllers/AuthorizationController.cs (hunk 2 above)
Comment (KovalenkoAndrii) [reply MyBooking/Controllers/AuthorizationController.cs]: Done

**Review by TanyaKalistruk — APPROVED** [review]

**Pull request merged**